ansible/README.md

```diff
@@ -0,0 +1,77 @@
+#Asqatasun Ansible Role
+This Ansible role will install the` Asqatasun Opensource Accessibility` on Ubuntu 12.04/14.04 LTS.
```

Comment:
Maybe we could add a few words to introduce Ansible purpose, or link to a tutorial (I found https://serversforhackers.com/an-ansible-tutorial which helped me to understand the big picture of Ansible)